app/assets/stylesheets/base/_tables.scss

```diff
@@ -0,0 +1,48 @@
+.table {
+  display: flex;
+  flex-flow: column nowrap;
+  font-size: .8rem;
+  margin: 0.5rem;
+  line-height: 1.5;
+  border-bottom: 1px solid #d0d0d0;
+  flex: 1 1 auto;
+}
+
+.th {
+  display: none;
+  font-weight: 700;
+  background-color: #f2f2f2;
+}
+
+.th > .td {
+  white-space: normal;
+  justify-content: center;
+}
+
+.tr {
+  width: 100%;
+  display: flex;
+  flex-flow: row nowrap;
+}
+
+.tr:nth-of-type(even) {
+  background-color: #f2f2f2;
```

Comment:
A suggestion to extract the colours into a colours library and use variables here?
That way, you define a specific shade of grey once, and reuse it everywhere.

Reply:
Done, thanks for the suggestion! 👌